test/audits/aria-role-not-scoped-test.js

```diff
@@ -79,3 +79,33 @@ test('Scope role missing', function() {
     equal(actual.result, axs.constants.AuditResult.FAIL);
     deepEqual(actual.elements, expected);
 });
+
+test('Scope role present - tablist', function() {
+    var rule = axs.AuditRules.getRule('ariaRoleNotScoped');
+    var fixture = document.getElementById('qunit-fixture');
+    var container = fixture.appendChild(document.createElement('ul'));
+    container.setAttribute('role', 'tablist');
+    for (var i = 0; i < 4; i++) {
+        var item = container.appendChild(document.createElement('li'));
+        item.setAttribute('role', 'tab');
+    }
+
+    var actual = rule.run({ scope: fixture });
+    equal(actual.result, axs.constants.AuditResult.PASS);
+    deepEqual(actual.elements, []);
+});
+
+test('Scope role missing - tab', function() {
+    var rule = axs.AuditRules.getRule('ariaRoleNotScoped');
+    var fixture = document.getElementById('qunit-fixture');
+    var expected = [];
+    for (var i = 0; i < 4; i++) {
+        var item = fixture.appendChild(document.createElement('span'));
```

Comment:
It was just to save one line - this test doesn't need a container, the spans are added directly to the fixture. It didn't feel right to add a bunch of `li` elements without a `ul` or `ol`.

I can change it to list if you prefer - the test will still work.